src/SFA.DAS.EmployerAccounts.Web.UnitTests/SFA.DAS.EmployerAccounts.Web.UnitTests.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <ProjectReference Include="..\SFA.DAS.EmployerAccounts.Web\SFA.DAS.EmployerAccounts.Web.csproj" />
     <ProjectReference Include="..\SFA.DAS.EmployerAccounts\SFA.DAS.EmployerAccounts.csproj" />
+    <ProjectReference Include="..\SFA.DAS.EmployerFinance\SFA.DAS.EmployerFinance.csproj" />
```

Comment:
Does this project reference need to remain or can it now be removed ?

Reply:
It can and has been removed